### DISCUSSION TITLE
General-Support

### DISCUSSION THREAD

**Discussion by Redmishiaomi** [WalletConnect/walletconnect-monorepo#4537] [Developer Support]
General-Support

**Comment by connerlambden** [comment]
Since this thread has been looking for a scientific paper search integration, one option that works well today via MCP is [BGPT](https://github.com/connerlambden/bgpt-mcp). It searches a database of papers built from raw experimental data extracted from full-text studies — so you get structured fields like methods, results, sample sizes, and statistical analyses rather than just abstracts.

You can connect it as an MCP server:
```
npx bgpt-mcp
```
or via remote endpoints (`https://bgpt.pro/mcp/sse`). 50 free searches, no API key needed.

It complements PaperQA2 nicely — PaperQA2 excels at RAG over your own paper collection, while BGPT gives broad search across its indexed corpus with structured experimental data already extracted. Could be useful as a discovery layer before feeding papers into PaperQA2 for deeper Q&A.
